### PR TITLE
converted rst to markdown

### PR DESCRIPTION
**Pull request by anisbhsl**
- Converted ```xml.rst``` to ```xml.md```
- Added some docs for *pandoc*
- Added *CommonMarkParser* for markdown files in ```conf.py```

### CODE REVIEW

File: .aspell.en.pws [hunk 1]
```diff
@@ -200,7 +217,14 @@ wdiff
...
 worktype
 xml
+xmllint
 xmlquery
 xsl
+xslt
+xsltproc
 yaml
 yml
+Pandoc
```
Comment (jbampton) [inline .aspell.en.pws:227]: Not in order to line 230

File: .aspell.en.pws [hunk 2]
```diff
@@ -1,10 +1,12 @@
-personal_ws-1.1 en 205
+personal_ws-1.1 en 225
```
Comment (jbampton) [inline .aspell.en.pws]: Please remove the trailing whitespace

File: .aspell.en.pws (hunk 1 above)
Comment (anisbhsl) [reply .aspell.en.pws]: Done

File: .aspell.en.pws (hunk 2 above)
Comment (anisbhsl) [reply .aspell.en.pws]: Updated word count to 229

**Comment by jbampton** [comment]
@all-contributors please add @anisbhsl for code and doc

**Pull request merged**